### PR TITLE
Added release notes for MW 1.3.64 and 1.3.65

### PR DESCRIPTION
**Pull request by TSchmiedlechner**

### CODE REVIEW

File: middleware/2024-10-10-1.3.64.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+# Middleware 1.3.64 (Germany)
+In this version of the Middleware, we've added the migration receipt for switching local installations to the CloudCashbox, and fixed an issue that prevented DSFinV-K exports on Queues that were initially set-up during the pilot phase in Germany.
+
+<!--truncate-->
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German and Italian markets only, customers in Austria and France should continue to use version 1.2. We are currently testing a unifyed version for all markets, and will release an update about this soon.
```
Comment (FlorianStadlberger) [inline middleware/2024-10-10-1.3.64.md:14]: *unified

File: middleware/2024-10-16-1.3.65.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+# Middleware 1.3.65 (Germany)
+In this version of the Middleware, we've made migrations from BYODC to the CloudCashbox easier by automatically redirecting requests, and fixed an issue that prevented TAR exports from TSEs on old Android devices from specific vendors.
+
+<!--truncate-->
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German and Italian markets only, customers in Austria and France should continue to use version 1.2. We are currently testing a unifyed version for all markets, and will release an update about this soon.
```
Comment (FlorianStadlberger) [inline middleware/2024-10-16-1.3.65.md:14]: *unified

File: middleware/2024-10-10-1.3.64.md (hunk 1 above)
Comment (TSchmiedlechner) [reply middleware/2024-10-10-1.3.64.md:14]: Great that nobody noticed this for 3 years :D

Comment (TSchmiedlechner) [reply middleware/2024-10-10-1.3.64.md]: Fixed, thanks

**Review by pawelvds — APPROVED** [review]

**Pull request merged**